BoxContentSDK/BoxContentSDK/Requests/BOXSearchRequest.m

```diff
@@ -159,6 +159,20 @@ - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBloc
     }
 }
 
+- (void)performRequestWithCached:(BOXItemArrayCompletionBlock)cacheBlock
+                       refreshed:(BOXItemArrayCompletionBlock)refreshBlock
+{
+    if (cacheBlock) {
+        if ([self.cacheClient respondsToSelector:@selector(retrieveCacheForFolderRequest:completion:)]) {
+            [self.cacheClient retrieveCacheForSearchRequest:self completionBlock:cacheBlock];
+        } else {
+            cacheBlock(nil, 0, NSMakeRange(0, 0), nil);
```

Comment:
is our convention to call the cacheBlock with an empty response, rather than just not calling the cacheBlock at all?